### PR TITLE
[LLM] Add examples for `gptneox`, `llama`, and `bloom` family model using transformers-like API

### PR DESCRIPTION
**Pull request by Oscilloscope98**
### 1. Why the change?

Add examples for `gptneox`, `llama`, and `bloom` family model using transformers-like API.

### 2. User API changes

N/A

### 3. Summary of the change 

- Add examples for `gptneox`, `llama`, and `bloom` family model using transformers-like API
- Add related README

### 4. How to test?
- [x] Local tests

### CODE REVIEW

File: python/llm/example/bloom/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```
+
+## Run Example
+```bash
+python ./gptneox.py --thread-num THREAD_NUM
+```
+arguments info:
+- `--thread-num THREAD_NUM`: required argument defining the number of threads to use for inference. It is default to be `2`.
+- `--repo-id-or-model-path REPO_ID_OR_MODEL_PATH`: optional argument defining the huggingface repo id from which the BLOOM family model is downloaded, or the path to the huggingface checkpoint folder for BLOOM family model. It is default to be `'bigscience/bloomz-7b1'`
+- `--promp PROMPT`: optional argument defining the prompt to be infered. It is default to be `'Q: What is AI? A:'`.
```
Comment (TheaperDeng) [inline python/llm/example/bloom/README.md:21]: any possible to add a `##sample output` section?

**Review by TheaperDeng — APPROVED** [review]
LGTM, please add sample output if it's easy to do

File: python/llm/example/bloom/README.md (hunk 1 above)
Comment (Oscilloscope98) [reply python/llm/example/bloom/README.md:21]: Have added the sample outputs for Inference :)

**Review by TheaperDeng — APPROVED** [review]
LGTM

**Pull request merged**